suites/portal/dataExplorerTest.js

```diff
@@ -0,0 +1,52 @@
+Feature('Data Explorer');
+
+Scenario('test exploring data using filters and sqon view', async (I) => {
```

Comment:
Good job covering all of the filter cases! I think we could simplify this a bit by defining the selector for the tabs and the items with XPath, then creating a function which selects the nth item.
e.g.
```
function clickNthItem(idx, xpath) {
  // XPATH is 1 base indexed, so increment by 1
  ++idx;
  this.click(`(${xpath})[idx]`);
}
let filter_selector = '//p[contains(@class, 'filter-group__tab')]'
// https://codecept.io/helpers/WebDriverIO/#grabnumberofvisibleelements
let n_filters = await I.grabNumberOfVisibleElements(filter_selector)
for (let i = 0; i<n_filters; ++i) {
  I.clickNthItem(i, filter_selector)
}
```
This might make things a little simpler since we wouldnt need the elements, elementId, elementIdClick functions. Haven't tested it though, so it may not work!

Reply:
Thanks! I will try to simplify this!